src/quacc/recipes/mlp/_base.py

```diff
@@ -64,6 +65,16 @@ def pick_calculator(
             kwargs["default_dtype"] = "float64"
         calc = mace_mp(**kwargs)
 
+    elif method.lower() == "sevennet":
+        from sevenn import __version__
+        from sevenn.sevennet_calculator import SevenNetCalculator
+
+        if "model" not in kwargs:
+            kwargs["model"] = "7net-0"
```

Comment:
7net-0 is the default model, so it seems like we should probably not specify this.

Reply:
I added both the default settings so the user knows what model and devices are being used if they want to dig deeper. But it's redundant, and I'll remove them in the next push.

Reply:
The default setting is "auto" for device I believe, but in any case let's remove them so we are always consistent with the underlying calculator.